src/main/resources/feedbackQuestionConstSumEditFormTemplate.html

```diff
@@ -37,15 +37,20 @@
                         <option value="true" id="constSumOption_Recipient-${questionNumber}" ${selectedConstSumPointsPerOption}>per recipient:</option>
                     </select>
                 </div>
-                <div class="col-sm-3">
+                <div class="col-md-3">
```

Comment:
Anything below the specified size, it assumes full width, i.e. 12 cols. This is Bootstrap's default behaviour valid everywhere.